src/pages/gen2/start/index.mdx

```diff
@@ -0,0 +1,237 @@
+export const meta = {
+  title: 'Getting Started',
+  description: 'Learn how to get started with AWS Amplify (Gen 2)'
+};
+
+## Installation
+
+Open your existing project or create a new one:
+
+<NodePackageManagerTabs
+  npm={`npm create next-app -- my-app --typescript\ncd my-app`}
+  yarn={`yarn create next-app my-app --typescript\ncd my-app`}
+  pnpm={`pnpm create next-app my-app --typescript\ncd my-app`}
+/>
+
+The easiest way to get started with AWS Amplify is through [npm](https://npmjs.com) with [`create-amplify`](https://www.npmjs.com/package/create-amplify)
+
+<NodePackageManagerTabs
+  npm="npm create amplify"
+  yarn="yarn create amplify"
+  pnpm="pnpm create amplify"
+/>
+
+You can run this command from within your existing project with little to no disturbances. It will scaffold a lightweight Amplify project with the following files:
+
+```text
+├── amplify/
+│   ├── auth/
+│   │   └── resource.ts
+│   ├── data/
+│   │   └── resource.ts
+│   ├── backend.ts
+│   └── package.json
+├── node_modules/
+├── .gitignore
+├── package-lock.json
+├── package.json
+└── tsconfig.json
+```
+
+Once scaffolded, you're ready to create your first cloud sandbox. Amplify gives every developer a personal cloud sandbox where they can test changes rapidly using live AWS services. Sandbox should not be used for production applications. To use Amplify's [Sandbox mode](http://localhost:3000#fix-me), run the following command:
+
+<NodePackageManagerTabs
+  npm="npx amplify sandbox"
+  yarn="yarn amplify sandbox"
+  pnpm="pnpm amplify sandbox"
+/>
+
+Try making a change to your data model and hit save. Changes are deployed in seconds.
+
+```ts {6} showLineNumbers
+import { type ClientSchema, a, defineData } from '@aws-amplify/backend';
+
+const schema = a.schema({
+  Todo: a.model({
+    title: a.string(),
+    description: a.string(),
+    priority: a.enum(['low', 'medium', 'high']).default('low')
+  })
+});
+
+export type Schema = ClientSchema<typeof schema>;
+
+export const data = defineData({
+  schema
+});
+```
+
+## Authentication
+
+Each of the `resource.ts` files is where you _define_ the corresponding backend resource:
```

Comment:
May add a comment within this line on what type of auth is defined here - this is showing set up with email/password auth - correct?

For example, "Each of the `resource.tx` files is where you _define_ the corresponding backend resource. Here we are adding authentication using email.

Reply:
Added more of an intro here